### PR TITLE
Add getSignupAssessment

### PR DESCRIPTION
**Pull request by racevedoo**
This adds the `getSignupAssessment` method to `IncogniaAPI`.

Some additions in networking classes were needed to support http GET method.

### CODE REVIEW

File: src/test/java/com/incognia/TokenAwareDispatcher.java [hunk 1]
```diff
@@ -43,6 +45,10 @@ public MockResponse dispatch(@NotNull RecordedRequest request) {
       assertThat(request.getHeader("Authorization")).isEqualTo("Bearer " + token);
       return new MockResponse().setResponseCode(200).setBody("{\"name\": \"my awesome name\"}");
     }
+    if ("/api/v2/onboarding".equals(request.getPath()) && "GET".equals(request.getMethod())) {
+      assertThat(request.getHeader("Authorization")).isEqualTo("Bearer " + token);
+      return new MockResponse().setResponseCode(200).setBody("{\"name\": \"my awesome name\"}");
```
Comment (joaoaab) [inline src/test/java/com/incognia/TokenAwareDispatcher.java:50]: I think we can refactor these body strings and paths to constants

Comment (joaoaab) [reply src/test/java/com/incognia/TokenAwareDispatcher.java:50]: Also i think that this method is getting a little bit out of hand with so much code :smile:
Do you think this is the right time to refactor the content of these ifs with methods? or should we wait a little more?

Comment (racevedoo) [reply src/test/java/com/incognia/TokenAwareDispatcher.java:47]: I guess I can refactor this in another PR

File: src/test/java/com/incognia/IncogniaAPITest.java [hunk 2]
```diff
@@ -100,4 +100,42 @@ void testRegisterSignup_whenNullAddress() {
...
+
+    Map<String, Object> expectedEvidence = new HashMap<>();
+    expectedEvidence.put("device_model", "Moto Z2 Play");
+    expectedEvidence.put("geocode_quality", "good");
+    expectedEvidence.put("address_quality", "good");
+    expectedEvidence.put("address_match", "street");
+    expectedEvidence.put("location_events_near_address", 38);
+    expectedEvidence.put("location_events_quantity", 288);
+    expectedEvidence.put("location_services", locationServices);
+    expectedEvidence.put("device_integrity", deviceIntegrity);
```
Comment (figueredo) [inline src/test/java/com/incognia/IncogniaAPITest.java:137]: nit: this could be moved to a `*Fixture` helper.

**Review by figueredo — APPROVED** [review]

**Review by joaoaab — APPROVED** [review]

**Pull request merged**